pkg/utils/validationUtils.go

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package utils
+
+// Checks if a given string contains only characters in the Latin-1 character set (codepoints 0-255),
```

Comment:
Shouldn't we dis-allow control codes like newline, tab, carriage-return, bell ..etc. eg: Latin-1 isn't restrictive enough I feel. Don't think they should have spaces inside, or a vertical bar, as that will mess up the formatting of the raw format output.

Reply:
See: https://cs.stanford.edu/people/miles/iso8859.html

Reply:
Replaced Latin-1 usage for group names with alphanumeric characters (a-z, A-Z, 0-9), dashes (-), and underscores (_) in https://github.com/galasa-dev/cli/commit/203965f1ba5c02f28eb123cea869ee0a9f47d054